src/System.Runtime.Extensions/src/System/Environment.Win32.cs

```diff
@@ -341,13 +342,17 @@ public static string UserName
                 }
 
                 return string.Empty;
+#else
+                return "Windows User";
+#endif
```

Comment:
@stephentoub I think it will be slightly confusing:
System\Environment.cs
System\Environment.Unix.cs
System\Environment.Windows.cs
System\Environment.Win32.NonUAP.cs
System\Environment.Win32.UAP.cs